ruby/import_js/importer.rb

```diff
@@ -164,11 +164,18 @@ def find_files(variable_name)
       if alias_path = @config.get('aliases')[variable_name]
         return [alias_path]
       end
+      dashed_variable = camelcase_to_dashed(variable_name)
       snake_case_variable = camelcase_to_snakecase(variable_name)
       matched_files = []
       @config.get('lookup_paths').each do |lookup_path|
         Dir.chdir(lookup_path) do
-          matched_files.concat(Dir.glob("**/#{snake_case_variable}.js*"))
+          matched_files.concat(
+            Dir.glob([
+              "**/#{dashed_variable}.js*",
+              "**/#{snake_case_variable}.js*",
+              "**/#{variable_name}.js*"
+            ])
```

Comment:
I ran some benchmarking to see the performance penalty of introducing three matchers to `glob`, and it looks like the perf penalty is linear, at least on a codebase of Brigade's size. I've already had some complaints about performance from @lencioni before (https://github.com/trotzig/import-js/issues/22), and I'm a little hesitant to merge this as-is. I totally understand and agree with the rationale behind this commit though, so I'll list a few options I can think of to have the same functionality without introducing too much performance debt.

1. Make filename convention configurable so that only one glob pattern has to be used.
2. Reduce the three matchers to two by using the `?` matcher. E.g. `Dir.glob(['**/icon?button.js*', '**/IconButton.js*'])`
3. Reduce the three matchers to one: `Dir.glob('**/icon*button.js*'). There's a targeting penalty with that though, as e.g. `/js/components/icon_alternative_button.jsx` will also be matched. That might not be a big deal though.

Here's a benchmark I ran:
```
irb(main):022:0> puts Benchmark.measure { Dir.glob('**/icon_button.js*') }
  0.040000   0.150000   0.190000 (  0.190436)
=> nil
irb(main):023:0> puts Benchmark.measure { Dir.glob(['**/icon_button.js*', '**/IconButton.js*', '**/icon-button.js*']) }
  0.110000   0.450000   0.560000 (  0.568228)
=> nil
```